src/DetailsView/handlers/allurls-permission-handler.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
minor comment here: should the file be called `all-urls-permisisons-handler.ts`? (dash between **all** and **urls**